src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1360,7 +1365,11 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
       complete: () => {},
     };
 
-    this.paymentRequestButton = this.stripeService.getPaymentRequestButton(donation, paymentRequestResultObserver);
+    if (! this.stripeElements) {
+      throw new Error("Stripe elements not ready");
+    }
+
+    this.paymentRequestButton = this.stripeService.getPaymentRequestButton(donation, this.stripeElements, paymentRequestResultObserver);
```

Comment:
Wallets seem to be switched on inside this PR's use of Payment so far. So shouldn't we also remove the PRB completely?

Or, if we want a smaller first step, we could turn off wallets in the new element for now.